### DISCUSSION TITLE
Email draft containing all available signatures

### DISCUSSION THREAD

**Discussion by rayakajamon** [Set-OutlookSignatures/Set-OutlookSignatures#128] [Q&A] [answered]
Using the paid Benefactor license we get the below functionality. Is there a way to disable this feature?

**Create an email draft containing all available signatures in HTML and plain text for easy access in mail clients that do not have a signatures API**

**Comment by GruberMarkus** [comment, marked as answer]
@rayakajamon,

your question is about a Benefactor Circle feature. Please contact [ExplicIT Consulting](https://explicitconsulting.at/open-source/set-outlooksignatures/), our partner creating the Benefactor Circle add-on, for support.
